src/scenes/home/families/families.css

```diff
@@ -0,0 +1,17 @@
+.quoteSection {
+  margin-top: 50vh;
+  position: relative;
+}
+
+@media screen and (max-width: 1700px) {
```

Comment:
If you go with resizing the font's based upon our `src/index.css` global styles, you will likely be able to get rid of this whole media query.

Reply:
Was able to get this effect by just removing this media query. So should be good.